tests/test_web/test_api_v1.py

```diff
@@ -2103,3 +2089,27 @@ def test_get_build_related_bundles_not_configured(client, db, minimal_request_re
     rv = client.get(f'/api/v1/builds/{request_id}')
     assert rv.status_code == 200
     assert 'related_bundles' not in rv.json
+
+
+@mock.patch('iib.web.api_v1.status')
+def test_get_status(mock_status, client):
+    mock_status.return_value = {'services': {}, 'workers': []}
+    rv = client.get('/api/v1/healthcheck/details')
+    assert rv.status_code == 200
+    assert rv.json == mock_status.return_value
+    mock_status.assert_called_once()
+
+
+@pytest.mark.parametrize('error', [None, 'something is wrong'])
+@mock.patch('iib.web.api_v1.status')
+def test_get_status_short(mock_status, error, client):
```

Comment:
Could we have tests for success and failure separated? The good practice is to use separate functions if you want to test `if` statement
- test_get_status_short_success
- test_get_status_short_failure

Reply:
aha. I'll keep that in mind when creating tests in the future!

Reply:
On the other hand, if you do it in this PR we do not have to modify it in a future, right?
You can have a look to this article: https://www.nerdwallet.com/blog/engineering/5-pytest-best-practices/ 
From the article:
> Parametrize the same behavior, have different tests for different behaviors
> 
> tl;dr: Parametrize when asserting the same behavior with various inputs and expected outputs. Make separate tests for distinct behaviors.

Correct me if I am wrong, but I think it is different behavior when test ends with error and when it is successful.

Reply:
Yeah, if it is that important, we can do it in the future! I think the tests in test_healthcheck file is enough. Plus, I don't have the capacity for further requests. :)

Reply:
@zanssa Could you please make the change on this suggestion. I agree success and failure should be separate tests.